plugins/flytekit-torch-elastic/flytekitplugins/torchelastic/task.py

```diff
@@ -13,8 +13,7 @@
 
 @dataclass
 class Elastic(object):
-    min_replicas: int = 1
-    max_replicas: int = 1
+    nnodes: typing.Union[int, str] = 1
```

Comment:
Let's use [the way torchrun exposes](https://pytorch.org/docs/stable/elastic/run.html#elastic-min-1-max-4-tolerates-up-to-3-membership-changes-or-failures) this to the user, e.g. `--nnodes 2:3`